<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="mainGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="11" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="animFrameRateLabel">
          <property name="text">
           <string>Animation frame rate (in frames per second)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fpsSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="12" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="includePathsLabel">
          <property name="text">
           <string>Include paths:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="includePathsLineEdit">
          <property name="text">
           <string>Examples/Include;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="13" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="exrBinPathsLabel">
          <property name="text">
           <string>EXR binaries paths:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="exrBinPathsLineEdit">
          <property name="text">
           <string>bin;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="14" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="stylesheetLabel">
          <property name="text">
           <string>Editor stylesheet:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stylesheetLineEdit">
          <property name="text">
           <string>font: 9pt Courier;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="15" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>348</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="moveMainCheckBox">
        <property name="toolTip">
         <string>For compatibility with some GPU's.</string>
        </property>
        <property name="text">
         <string>Move main() to end</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="autoRunCheckBox">
        <property name="toolTip">
         <string>Disabling this might make debugging easier.</string>
        </property>
        <property name="text">
         <string>Autorun On Load</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="drawGLPathsCheckBox">
        <property name="toolTip">
         <string>Some engines use non GL matrices.</string>
        </property>
        <property name="text">
         <string>Draw spline paths for camera and target</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="splineOccCheckBox">
        <property name="text">
         <string>Spline Occlusion</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="lineNumbersCheckBox">
        <property name="toolTip">
         <string>Enabling this might be helpful when debugging.</string>
        </property>
        <property name="text">
         <string>Add line numbers to the script editing window.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="loopPlayCheckBox">
        <property name="text">
         <string>Loop animation when playing</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="saveEasingCheckBox">
        <property name="text">
         <string>Presets contain easing curve settings</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="useDefinesCheckBox">
        <property name="text">
         <string>Locked uniforms use #define or const</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>328</width>
          <height>32</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="refreshRateLabel">
          <property name="text">
           <string>OpenGL refresh rate (in milliseconds)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="refreshRateSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="logToFileCheckBox">
        <property name="toolTip">
         <string>Log to file instead of console, may require app restart</string>
        </property>
        <property name="text">
         <string>Log to File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
